lib/sycamore/sycamore/transforms/extract_entity.py

```diff
@@ -170,6 +171,7 @@ def __init__(
         self,
         child: Node,
         entity_extractor: EntityExtractor,
+        context: Optional[Context] = None,
         **resource_args,
     ):
-        super().__init__(child, f=entity_extractor.extract_entity, **resource_args)
+        super().__init__(child, f=entity_extractor.extract_entity, f_args={"context": context}, **resource_args)
```

Comment:
I don't understand why you had to add f_args.
It looks like from the code you could have written:
```
super().__init__(child, f=entity_extractor.extract_entity, context=context, **resource_args)
```
The kwargs are being passed into the function already.  At a minimum they should be called f_kwargs, (f_args would be the positional ones), but I don't think they're needed.

I think you could have also written:
```f=lambda d: entity_extractor.extract_entity(d, context=context)```

Reply:
Discussed offline, going to use a the 'kwargs' that already exists in BaseMap. Will update docstrings to clarify they exist in Map